caluma_interval/interval.py

```diff
@@ -0,0 +1,117 @@
+import logging
+from datetime import date, timedelta
+
+from envparse import env
+from isodate import parse_date, parse_duration
+
+from caluma_interval.client import CalumaClient
+from caluma_interval.queries import intervalled_forms_query, start_case_mutation
+
+logger = logging.getLogger(__name__)
+
+
+__title__ = "caluma_interval"
+__description__ = "Caluma companion app for handling intervalled forms"
+__version__ = "1.0.0"
+__author__ = "Adfinis SyGroup"
+
+
+class IntervalManager(object):
```

Comment:
No need to inherit from object in python3 anymore :)

Reply:
Done!